### PR TITLE
Added description for server

### PR DESCRIPTION
**Pull request by Dashboy1998**
## Context 

* Add a description to the server

## Choices

* Implemented the same as the other settings

## Test instructions

1. Set a description in docker-compose
2. Start container 
3. Join game
4. Go to the options in game to see the description. For my testing I did `Hello World`.

## Checklist before requesting a review

* [x] I have performed a self-review of my code
* [x] I've added documentation about this change to the README.
* [x] I've not introduced breaking changes.

### CODE REVIEW

**Comment by fryfrog** [comment]
You should wait for https://github.com/thijsvanloef/palworld-server-docker/pull/124 to be merged, then update your changes to incorporate it. The current way does not support spaces in the server name and wouldn't in the description.

**Comment by Dashboy1998** [comment]
> You should wait for https://github.com/thijsvanloef/palworld-server-docker/pull/124 to be merged, then update your changes to incorporate it. The current way does not support spaces in the server name and wouldn't in the description.

I didn't have any issues with spaces in the server name or description as and that can be seen in the photo in the PR.

**Review by thijsvanloef — CHANGES_REQUESTED** [review]
I've just merged #124, if you are able to, please change the way -serverdescription gets added to the STARTCOMMAND.

**Comment by thijsvanloef** [comment]
Needed to revert #124, your PR is still on my radar, but #124 has to be fixed first, sorry for the inconvenience.

**Comment by thijsvanloef** [comment]
There was some fuckery going on with the revert, I believe I fixed it

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@ services:
          - TZ=UTC
          - ADMIN_PASSWORD="adminPasswordHere"
          - COMMUNITY=false  # Enable this if you want your server to show up in the community servers tab, USE WITH SERVER_PASSWORD!
+         # Enable the environment variables below if you have COMMUNITY=true
```
Comment (Dashboy1998) [inline docker-compose.yml:21]: Oops I added this comment again.

**Pull request merged**